Backend-Type-Annotations.md

```diff
@@ -18,9 +18,18 @@ Mypy checks script (`scripts/run_mypy_checks.py`) is the script used to run our
 
 It has two modes of running:
 
-1. `python -m scripts.run_mypy_checks`\
+1. Python:
+  `python -m scripts.run_mypy_checks`\
+
+   Docker:
+   `make run_tests.mypy`
```

Comment:
Write all commands in triple backticks (```)

Reply:
ok